src/builder.rs

```diff
@@ -40,7 +40,7 @@ use crate::{error::*, Bytes, Uuid, Variant, Version};
 /// ```
 #[allow(missing_copy_implementations)]
 #[derive(Debug)]
-pub struct Builder(Uuid);
+pub struct Builder(pub Uuid);
```

Comment:
Was making this field public intentional?

Reply:
Yeah.. this was necessary due to moving modules around. actually.. it could probably be pub (crate) if

Reply:
ok.. I just checked and maybe it's not necessary.   Removed.